xcs-ui/xcs-ui-mainwikipreferences/pom.xml

```diff
@@ -94,6 +94,20 @@
               <xpath>/xwikidoc/object/property/version</xpath>
               <value>${xcs.footer.version}</value>
             </transformation>
+            <!-- Specify different rights for XWiki.XWikiGuest -->
+            <transformation>
+              <artifact>org.xwiki.enterprise:xwiki-enterprise-ui-admin-user</artifact>
+              <file>XWiki/XWikiPreferences.xml</file>
+              <xpath>xwikidoc/object[property/users[text()="XWiki.XWikiGuest"]]/property/levels</xpath>
+              <value>none</value>
```

Comment:
Actually, after a closer look, it seems it is complicated to use both the transformation plugin and the replacer one. They are not supposed to be used together. So let's go with the current PR, and we may look at this for the next release.